homeassistant/components/pcf8574/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "i2c_port_num": "I2C Port Number",
+          "i2c_address": "I2C Address",
+          "switch_1_name": "Switch 1",
+          "switch_2_name": "Switch 2",
+          "switch_3_name": "Switch 3",
+          "switch_4_name": "Switch 4",
+          "switch_5_name": "Switch 5",
+          "switch_6_name": "Switch 6",
+          "switch_7_name": "Switch 7",
+          "switch_8_name": "Switch 8",
+          "invert_logic": "Invert Logic"
+        }
+      }
+    },
+    "error": {
+      "ioerror": "cannot access i2c device",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
Can a unique_id be set, if not removed the already_configured string.
https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#unique-ids

Reply:
I'm using config entry id as unique_id. Can I use that for the already_configured check ? 
If not I'd just remove that string

Reply:
Can a combination of i2c_address and i2c_port_num be used? Is this unique?

> I'm using config entry id as unique_id. Can I use that for the already_configured check ?

This comment is about the ConfigFlow unique_id not an Entity unique_id.
You can't use a config_entry_id because this is random and generated after the config_flow.

Reply:
thanks. that is a good idea. I changed everything in 6abcca8e4f5436fa42e39840a841e799978b19c4 and now the error message is displayed when you try to create a second integration with the same `i2c_address`and `i2c_port_num`